devtools/generate_supported.py

```diff
@@ -29,10 +29,12 @@ class SupportedVersion(NamedTuple):
     DeviceType.StripSocket: "Power Strips",
     DeviceType.Dimmer: "Wall Switches",
     DeviceType.WallSwitch: "Wall Switches",
+    DeviceType.Fan: "Wall Switches",
     DeviceType.Bulb: "Bulbs",
     DeviceType.LightStrip: "Light Strips",
     DeviceType.Hub: "Hubs",
     DeviceType.Sensor: "Sensors",
+    DeviceType.Thermostat: "Sensors",
```

Comment:
Perhaps this should be its own category, or alternatively, we could rename "Sensors" to "Hub-connected devices"/"Devices requiring a hub"?

Reply:
Done